tests/unit/test-loader-test.js

```diff
@@ -3,7 +3,11 @@ import { module, test } from 'qunit';
 
 module('Unit | test loader', {
   beforeEach() {
-    this.TestLoader = window.require('ember-cli/test-loader')['default'];
+    try {
+      this.TestLoader = window.require('ember-cli-test-loader/test-support/index')['default'];
```

Comment:
is it possible for us to avoid the try/catch.

1. it will swallow legit errors as written
2. it will add another break when break on all exceptions (hurting debugging)
3. we have `require.entries` which can be used for existence checks.